CDCgov/NEDSS-Modernization
# CNFT2-1223 legacy custom fields banner

## Description

A link to the `Legacy Custom Fields Admin` page needs to be added to the Page Library

## Tickets

* [CNFT2-1223](https://cdc-nbs.atlassian.net/browse/CNFT2-1223)

## Steps to verify

1. Navigate to http://localhost:8080/nbs/login
2. Log in as `superuser`
3. Click System Management -> Page Management -> Manage Pages
4. Verify `Custom Fields Admin` banner is displayed
5. Click the link within the banner
6. Verify Classic™ custom fields admin screen is loaded


![legacyCustomAdmin](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/50c8220e-f46c-4b97-af1b-5fd07716e219)


[CNFT2-1223]: https://cdc-nbs.atlassian.net/browse/CNFT2-1223?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
